MediaContentSynchronization/Model/Publish.php

```diff
@@ -0,0 +1,47 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\MediaContentSynchronization\Model;
+
+use Magento\Framework\MessageQueue\PublisherInterface;
+
+/**
+ * Publish media content synchronization queue.
+ */
+class Publish
+{
+    /**
+     * Media content synchronization queue topic name.
+     */
+    private const TOPIC_MEDIA_CONTENT_SYNCHRONIZATION = 'media.content.synchronization';
+
+    /**
+     * @var PublisherInterface
+     */
+    private $publisher;
+
+    /**
+     * Image constructor.
```

Comment:
I'd avoid using the comments for constructor's phpdocs unless it's really necessary. I think they are not useful to developers and lead only to the copy-paste or renaming mistakes like here.